### ISSUE TITLE
user login and accounts

### ISSUE THREAD

**Issue by RustoMCSpit** [open-audio-stack/open-audio-stack-registry#24]
stuff like the below

https://tonehunt.org/emil.rohbe
![image](https://github.com/user-attachments/assets/c6ecfadc-4286-4825-9218-9c48a62a21c7)
![image](https://github.com/user-attachments/assets/05e804ad-2d33-413b-8a57-ef1e6228c5db)
![image](https://github.com/user-attachments/assets/79ba42d5-8a5b-4c22-8958-5148495e6186)

**Comment by kmturley** [comment]
Author/org profile is a good idea.

In StudioRack I made a landing page for all the plugins from a single author/org:
https://studiorack.github.io/studiorack-site/instruments/ryukau
https://studiorack.github.io/studiorack-site/instruments/freepats

Would be awesome to make that actually a full profile page with more about the author/org.

**Comment by RustoMCSpit** [comment]
https://github.com/open-audio-stack/open-audio-stack-registry/blob/main/src/projects/kmt/banwer/1.0.1/index.yaml

how will author KMT be assigned a profile? is KMT an account name that can be logged into down the line? if so, how do we prevent others from uploading things as KMT in a .yaml file without having to manually check each upload?

this also applies to presets and projects. will we just email each plugin developer their account details when this feature becomes available?

**Comment by RustoMCSpit** [comment]
this would require cookies, meaning a cookie warning would need to be added

**Comment by kmturley** [comment]
Open Audio Stack is currently a static site, so there is not a way to have features such as:
- Accounts
- Logins
- Save/update database

At the moment I don't want to change architecture as it will incur costs and introduce bandwidth and security risks.

However we can still support a static profile page which is generated from a yaml file, just like the package metadata.

**Comment by RustoMCSpit** [comment]
> Open Audio Stack is currently a static site, so there is not a way to have features such as:
> 
>     * Accounts
> 
>     * Logins
> 
>     * Save/update database
> 
> 
> At the moment I don't want to change architecture as it will incur costs and introduce bandwidth and security risks.
> 
> However we can still support a static profile page which is generated from a yaml file, just like the package metadata.

the way around this is to make a dynamic site that leads into a static site.

**Comment by kmturley** [comment]
Yes could have a dynamic site which produces a static feed, but would still require more work and funding for hosting costs. I would prefer to keep Open Audio Stack simple and static for now, while the standard is being adopted.

There is nothing stopping you from building this separately. It's possible for other registries which implement the same the standards to use server-side code, the standard mentions it. If you want those features you could build a registry which could be added and linked which is built using a dynamic stack.

**Comment by RustoMCSpit** [comment]
By 'produce' you mean re-directs you, right? I am suggesting a dynamic site to handle accounts and logins that then re-directs you to a static one.

**Comment by kmturley** [comment]
Yes with a server-side site you can choose which paths hit server-side code, and which hit static files.
It would be possible to have a hybrid where when data is changed, the static feed would be regenerated.
